app/src/main/java/org/oppia/android/app/shim/ViewBindingShimImpl.kt

```diff
@@ -63,6 +65,27 @@ class ViewBindingShimImpl @Inject constructor() : ViewBindingShim {
     return DataBindingUtil.findBinding<ProfileInputViewBinding>(profileInputView)!!.errorText
   }
 
+  override fun inflatePromotedStoryCardBinding(
+    inflater: LayoutInflater,
+    parent: ViewGroup,
+    attachToParent: Boolean
+  ): View {
+    return PromotedStoryCardBinding.inflate(
+      LayoutInflater.from(parent.context),
+      parent,
+      /* attachToParent= */ false
```

Comment:
Should this be the ``attachToParent`` variable, instead?

Reply:
Done -- the other implementations set it to false so I was just following that but I agree that it doesn't really make sense.